procedures/katello/release.md.erb

```diff
@@ -26,12 +26,13 @@
   * [ ] Set version to `<%= version %>`
   * [ ] Set prerelease to `<%= extra %>`
 <% unless is_rc %>  * [ ] Reset release to `1`
-<% end %>  * [ ] In each package dir, remove the old tarball, run `spectool -g *.spec` and `git annex add *.tar.gz` or `git annex add *.gem`
+<% end %>  * [ ] Fetch new source by running `obal source rubygem-katello`, then `git rm` the old source.
   * [ ] Edit `katello-repos/katello.repo` to set `gpgcheck=1`
-  * [ ] Commit with message "Release <%= full_version %>"
+  * [ ] Ensure `katello-repos/RPM-GPG-KEY-katello` has the right key for the release
```

Comment:
In Foreman we do this as part of branching. Perhaps update the Foreman branching instructions to also update Katello?

Reply:
If we are moving to the same GPG key signing all, would we still need this present here?

Reply:
I think ideally we would build katello-repos from the foreman-release source, then all of this is obsolete. Today we ship the (foreman) key in katello-repos, so that's what's been documented (for the lack of time to properly think through the newer process).